site/docs/index.md

```diff
@@ -5,62 +5,34 @@ title: Home
 
-
-## Example Use Cases
-
-* Communicate a compute plan between a SQL parser and an execution engine (e.g. Calcite SQL parsing to Arrow C++ compute kernel)
-* Serialize a plan that represents a SQL view for consistent use in multiple systems (e.g. Iceberg views in Spark and Trino)
-* Submit a plan to different execution engines (e.g. Datafusion and Postgres) and get a consistent interpretation of the semantics.
-* Create an alternative plan generation implementation that can connect an existing end-user compute expression system to an existing end-user processing engine (e.g. Pandas operations executed inside SingleStore)
-* Build a pluggable plan visualization tool (e.g. D3 based plan visualizer)
-
-
-
-## Community Principles
-
-* Be inclusive and open to all. If you want to join the project, open a PR or [issue](https://github.com/substrait-io/substrait/issues), start a [discussion](https://groups.google.com/g/substrait) or [join the Slack Channel]({{versions.slackinvitelink}}).
-* Ensure a diverse set of contributors that come from multiple data backgrounds to maximize general utility.
-* Build a specification based on open consensus.
-* Avoid over-reliance/coupling to any single technology.
-* Make the specification and all tools freely available on a permissive license (ApacheV2)
-
+We highly recommend the [tutorial](/tutorial/sql_to_substrait/) to learn how a Substrait plan is constructed.
 
 
-## Related Technologies
 
-* [Apache Calcite](https://calcite.apache.org/): Many ideas in Substrait are inspired by the Calcite project. Calcite is a great JVM-based SQL query parsing and optimization framework. A key goal of the Substrait project is to expose Calcite capabilities more easily to non-JVM technologies as well as expose query planning operations as microservices.
-* [Apache Arrow](https://arrow.apache.org/): The Arrow format for data is what the Substrait specification attempts to be for compute expressions. A key goal of Substrait is to enable Substrait producers to execute work within the Arrow Rust and C++ compute kernels.
+## Benefits
 
+* Avoids every system needing to create a communication method between every other system -- each system merely supports ingesting and producing Substrait and it instantly becomes a part of the greater ecosystem.
+* Makes every part of the system upgradable.  There's a new query engine that's ten times faster?  Just plug it in!
+* Enables heterogeneous environments -- run on a cluster of an unknown set of execution engines!
+* The text version of the Substrait plan allows you to quickly see how a plan functions without needing a visualizer (although there are Substrait visualizers as well!).
```

Comment:
Should we hold off on mentioning a visualizer until we can provide a link/more information on it?

Reply:
One of my pending PRs has a link to Substrait Fiddle which should cover that.